src/prefect/cli/key_value.py

```diff
@@ -0,0 +1,109 @@
+import click
+
+from prefect import Client, config
+
+
+@click.group(hidden=True)
```

Comment:
I'm not sure I understand why we so often hide then reproduce the default click output. I'd use Jim's work at `build_register` as best-practice instead.